src/sentry/integrations/mixins/repositories.py

```diff
@@ -60,6 +62,8 @@ def check_file(self, repo: Repository, filepath: str, branch: str) -> str | None
         except ApiError as e:
             if e.code != 404:
                 raise
+
+            sentry_sdk.capture_exception()
```

Comment:
This may become a bit noisy if a good number of customers have broken/outdated integrations so I can replace this with logging if this occurs.

Reply:
@GabeVillalobos are you looking to capture 404 api errors here?

Reply:
One thing to watch out for if there some errors are expected - it may trip canary during deploys

Reply:
You'll only be getting 404's here which are the least interesting errors I think. Should this be on 64 instead?